### PR TITLE
Install debian package requirements via sdw-config

### LINKED ISSUE #1956: Install packages required on debian-12-minimal base template via securedrop-workstation-config dependencies
Currently the following Salt is used to install requirements not present in `debian-12-minimal`:

```
sd-base-template-install-additional-packages:                                   
  pkg.installed:                                                                
    - pkgs:                                                                     
      - qubes-core-agent-passwordless-root                                      
      - rsyslog                                                                 
      - mailcap                                                                 
      - apparmor                                                                
      - apparmor-utils     
``` 

Since they are required for all appVMs, we could install them via the securedrop-workstation-config package instead.

### PR DESCRIPTION
**Pull request by zenmonkeykstop**
## Status

Ready for review 
## Description

Fixes #1956

## Test Plan

- [x] CI is passing
- [x] `DEBIAN_VERSION='bookworm' make build-debs` succeeds
- [ ] Installing resultant securedrop-workstation-config package on a debian-12-minimal template pulls in qubes-core-passwordless-root, rsyslog, mailcap, apparmor, apparmor-utils, nautilus.
- [x] Visual review looks good.

## Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:

 - [ ] I have tested these changes in the appropriate Qubes environment
 - [ ] I do not have an appropriate Qubes OS workstation set up (the reviewer will need to test these changes)
 - [ ] These changes should not need testing in Qubes

If these changes add or remove files other than client code, the AppArmor profile may need to be updated. Please check as applicable:

 - [ ] I have updated the [AppArmor profile](https://github.com/freedomofpress/securedrop-client/blob/HEAD/files/usr.bin.securedrop-client)
 - [ ] No update to the AppArmor profile is required for these changes
 - [ ] I don't know and would appreciate guidance

If these changes modify the database schema, you should include a database migration. Please check as applicable:

 - [ ] I have written a migration and upgraded a test database based on `main` and confirmed that the migration is [self-contained] and applies cleanly
 - [ ] I have written a migration but have not upgraded a test database based on `main` and would like the reviewer to do so
 - [ ] I need help writing a database migration
 - [ ] No database schema changes are needed

[self-contained]: https://github.com/freedomofpress/securedrop-client#generating-and-running-database-migrations

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -41,7 +41,7 @@ Description: This is securedrop Qubes proxy service
 
 Package: securedrop-workstation-config
 Architecture: all
-Depends: nautilus, securedrop-keyring
+Depends: qubes-core-passwordless-root, rsyslog, mailcap, apparmor, apparmor-utils, nautilus, securedrop-keyring
```
Comment (legoktm) [inline debian/control:44]: What is apparmor-utils needed for? I had removed it from the client in https://github.com/freedomofpress/securedrop-client/pull/1856.

**Review by legoktm — COMMENTED** [review]
I think this is fine for now, but we should be looking to remove passwordless-root IMO (only needed by print stuff AFAICT). Have a question about apparmor-utils.

File: debian/control (hunk 1 above)
Comment (zenmonkeykstop) [reply debian/control:44]: I *thought* it was needed for `aa-status` but looks like that's in the `apparmor` package.

Comment (legoktm) [reply debian/control:44]: Yeah, https://manpages.debian.org/bookworm/apparmor-utils/index.html is the commands that it provides, I don't see any we need?

**Review by legoktm — APPROVED** [review]
Local build is just finishing up, but LGTM

**Pull request merged**